code/game/objects/structures/mirror.dm

```diff
@@ -143,11 +143,11 @@
 	if(istype(get_area(src),/area/merchant_station))
 		if(istype(user) && user.mind && user.mind.assigned_role == "Merchant" && user.species.name != "Vox")
 			var/choice = input("Do you wish to become a Vox? This is not reversible.") as null|anything in list("No","Yes")
-			if(choice && choice == "Yes")
+			if(choice == "Yes")
 				var/mob/living/carbon/human/vox/vox = new(get_turf(src),"Vox")
 				if(user.mind)
 					user.mind.transfer_to(vox)
-				spawn(1)
+

```

Comment:
Deleting this without changing the indentation has changed the meaning of the below block.